src/Codeception/Lib/InnerBrowser.php

```diff
@@ -2030,7 +2054,7 @@ protected function setCookiesFromOptions()
     }
 
     /**
-     * @return string
+     * @return string|null
```

Comment:
preg_replace only returns NULL if pattern is invalid.
The pattern is very simple and hardcoded right here. I don't think that it can possibly return NULL.